ui/src/app/dashboard/services/DashService.js

```diff
@@ -0,0 +1,32 @@
+(function(){
+  'use strict';
+
+  angular.module('app')
+        .service('DashService', [
+        '$http',
+      dashService
+  ]);
+
+  function dashService($http){
+    var addDownload = function(link) {
+      return $http({
+          method: 'POST',
+          url: 'http://localhost:5000/api/download',
+          data: JSON.stringify(link),
+          headers: {'Content-Type': 'application/x-www-form-urlencoded'}
+      });
+    };
+
+    var getDownloads = function() {
+      return $http({
+        method: 'GET',
+        url: 'http://localhost:5000/api/downloads/1'
```

Comment:
Shall we update the api and change download statuses into words instead of numbers? It will increase the readability don't you think @paarandika ?